### DISCUSSION TITLE
Did Spotify Kill Librespot?

### DISCUSSION THREAD

**Discussion by cpttater** [librespot-org/librespot#1209] [General]
Today all of my clients stopped passing audio. No changes have been made on my end. Spotify App shows the streams and connects, but the stream remains idle. I have checked my firewall logs with no answers. 

Anyone else having an issue?

**Comment by cpttater** [comment]
It looks like I am receiving data from Spotify's IP address. Not sure what is going on here. 

![2023-10-18_16-07-23](https://github.com/librespot-org/librespot/assets/67788882/383dfd87-e259-4da3-adff-7a41d30c99a8)

**Comment by cpttater** [comment]
> Today all of my clients stopped passing audio. No changes have been made on my end. Spotify App shows the streams and connects, but the stream remains idle. I have checked my firewall logs with no answers.
> 
> Anyone else having an issue?

I have tested with a local .wav file in Snapcast and it works just fine.

**Comment by cpttater** [comment]
Here is my config in SnapCast that has been working flawlessly for months. 
![Snag_22587bda](https://github.com/librespot-org/librespot/assets/67788882/05538e60-61ef-49ef-8d6b-69a43c4cf9cf)

**Comment by cpttater** [comment]
Changed my config to the following and am now getting these errors. 

![2023-10-18_17-03-34](https://github.com/librespot-org/librespot/assets/67788882/3ebde2af-ac81-40fc-84c5-28ae24e89c26)

![2023-10-18_16-59-53](https://github.com/librespot-org/librespot/assets/67788882/d5e922bf-e4fb-4414-a284-103029c582d6)

**Reply by kingosticks** [reply]
If you can provide a standalone (without snapcast) recipe, including a verbose log and the actual tracks you are trying to play then it can be investigated. This level of redaction is not helpful.

**Reply by cpttater** [reply]
It is any track ID, I was redacting the playlist id as it is a private playlist....but here is a public track ID.

![2023-10-18_17-19-42](https://github.com/librespot-org/librespot/assets/67788882/85f96a78-8c41-412c-b87a-1d84ea64fb6b)

**Reply by kingosticks** [reply]
And your librespot version is?

**Reply by kingosticks** [reply]
It would be great if you could provide text rather than images. I can't copy and paste from an image.

**Reply by cpttater** [reply]
How do I find the version of librespot?

Oct 18 21:09:33 snapserver snapserver[1505]: Loading <Bad Medicine> with Spotify URI <spotify:track:72hcFp4tYkd3dbNA9dZ3Pv>

Oct 18 21:09:33 snapserver snapserver[1505]: Unable to load encrypted file: Error { kind: FailedPrecondition, error: ParseIntError { kind: InvalidDigit } }

Oct 18 21:09:56 snapserver snapserver[1505]: Loading <Bad Medicine> with Spotify URI <spotify:track:72hcFp4tYkd3dbNA9dZ3Pv>

**Reply by cpttater** [reply]
Version

![2023-10-18_17-28-13](https://github.com/librespot-org/librespot/assets/67788882/05f35ee0-4adf-40fe-a5fe-0810770159de)

**Reply by roderickvd** [reply]
That’s not librespot. From the ParseIntError I know it must be dev but we don’t know which commit. 

Please launch librespot with  --verbose and provide the full log that was requested before, redacting only username and IP addresses that you want to protect. 

The verbose log will contain the input / network traffic that apparently cannot be parsed as integer. It could well be that Spotify changed this input to now also contain other characters. In that case the parsing needs to be adapted.

**Comment by MergingMonkey** [comment]
I had the same problem starting yesterday evening. Re-building from the latest `dev` branch fixed it for me.

**Reply by cpttater** [reply]
Thanks for the direction...fixed it for me.

**Comment by cpttater** [comment]
This was corrected by recompiling Librespot and moving the newly compiled instance to /usr/local/bin and reloading Snapcast.
